README.md

```diff
@@ -55,7 +61,18 @@ To get the source compiled with desired custom code page used for string literal
 ```
 GCONV_PATH=../gconv gcc -fexec-charset=ruthyn -c -o main.o main.c
 ```
-where `../gconv` is the directory the custom module is installed in and `ruthyn` is the name or alias of the custom code page implemented by the module.
+where `../gconv` is the directory the custom module is installed in
+and `ruthyn` is the name or alias of the custom code page implemented
+by the module.
+
+Note that the gconv files are not needed at run-time, only compilation.
```

Comment:
Mmmm, I understand you want to guide others to properly understand the subject. But, frankly, this sentence sounds strange. Probably somewhere at top doing an introduction must be made a stress that this is a plug-in for `gcc`. And, if so, it will be obvious for the reader that neither `gcc` nor its plug-ins are used at run time. Moreover, normally this approach is used for MCUs firmware where definitely will be no any files of such a sort...
Please consider to remove this sentence and either add some wording at introduction guiding that the matter is of `gcc`'s plug-in or leave this for me for further updates to the readme.

Reply:
I will remove it and leave it to your further readme improvements. 

Yes, you are correct, the sentence is a bit strange. I added it because that was my biggest confusion. The official documentation for [gconv](https://www.gnu.org/software/libc/manual/html_node/glibc-iconv-Implementation.html) only talks about using such modules at runtime, so I had presumed that was the only way they could be used. I had not known about `-fexec-charset` before.